modules/wikipedia/index.js

```diff
@@ -2,21 +2,37 @@ const wiki = require('wikijs').default;
 
 module.exports.commands = ['wiki', 'wikipedia'];
 
-function prettyPage(page) {
+const urlRegex = /^(?:https?:\/\/)?(?:en\.)?wikipedia\.org\/wiki\/(.+)/;
```

Comment:
I believe this regex won't be correct in a number of cases.

The most obvious is fragments, for example `https://en.wikipedia.org/wiki/Cgroups#Unified_hierarchy` will capture `#unified_hierarchy` as being part of the title.

Same for media urls like https://en.wikipedia.org/wiki/Gentoo_Linux#/media/File:Gentoo_Linux_logo_matte.svg

I would also much prefer having a host whitelist, as is done in the [amazon](https://github.com/euank/EuIrcBot/blob/79e49aa878e7c91ceb3f3d05617dcddaa5dab185/modules/amazon/index.js#L4) module, since that's less fragile than a regex IMO.